cli/api/interfaces.go

```diff
@@ -126,4 +126,6 @@ type API interface {
 	RemovePublicEndpointPort(serviceid, endpointName, portAddr string) error
 	EnablePublicEndpointPort(serviceid, endpointName, portAddr string, isEnabled bool) error
 	AddPublicEndpointVHost(serviceid, endpointName, vhost string, isEnabled, restart bool) (*servicedefinition.VHost, error)
+
+	EnablePublicEndpointVHost(serviceid, endpointName, vhost string, isEnabled bool) error
```

Comment:
There are places using VHost or Vhost. We might want to make it consistent for future maintenance.
I saw Vhost used in more places.